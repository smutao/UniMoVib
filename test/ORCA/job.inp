a test job

 $contrl
   qcprog="orca"
 $end

 $qcdata
   fchk="h2o.hess"
 $end
