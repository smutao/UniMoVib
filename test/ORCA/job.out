 =======================================================================================================================
       _/     _/                _/     _/            _/     _/
      _/     _/            _/  _/|  _/_/            _/     _/  _/  _/        UniMoVib:          A unified interface for
     _/     _/   _/_/_/       _/ |_/ _/   _/_/_/   _/     _/      _/           molecular harmonic vibrational frequency
    _/     _/  _/    _/  _/  _/  _/ _/  _/    _/  _/     _/  _/  _/_/_/        calculations.
   _/     _/  _/    _/  _/  _/     _/  _/    _/  _/    _/   _/  _/   _/
  _/     _/  _/    _/  _/  _/     _/  _/    _/   _/  _/    _/  _/   _/       Ver. 1.0.0, MAY 02, 2017
   _/_/_/   _/    _/  _/  _/     _/   _/_/_/     _/_/     _/  _/_/_/         https://github.com/zorkzou/UniMoVib
 =======================================================================================================================


 QC Program:      ORCA
 FCHK file:       h2o.hess


 Cartesian coordinates (Angstrom)
 ------------------------------------------------------------------------------------------
   No.   Atom    ZA                 X             Y             Z                Mass
 ------------------------------------------------------------------------------------------
     1    O       8            0.00000000    0.00000000    0.06290380           15.99900000
     2    H       1            0.00000000    0.78365557   -0.54376800            1.00800000
     3    H       1            0.00000000   -0.78365557   -0.54376800            1.00800000
 ------------------------------------------------------------------------------------------


 <<< SYMMETRY >>>
 Molecular Point Group (Z)   :   C2v 
 Molecular Point Group (Z+M) :   C2v 


 ************************************
 ***  Properties of Normal Modes  ***
 ************************************

 Results of vibrations:
 Normal frequencies (cm**-1), reduced masses (AMU),
 force constants (mDyn/A), IR intensities (km/mol)

                                                   1                                 2                                 3
          Irreps                                  A1                                A1                                B2
     Frequencies                           1698.5052                         3498.4302                         3642.8225
  Reduced masses                              1.0898                            1.0390                            1.0820
 Force constants                              1.8523                            7.4921                            8.4600
  IR intensities                             44.5622                            8.0730                            0.0038
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.00000  -0.00000   0.07386       0.00000   0.00000   0.04546       0.00000   0.07028   0.00000
           2   1         0.00000  -0.39207  -0.58613       0.00000   0.60730  -0.36077       0.00000  -0.55775   0.43179
           3   1         0.00000   0.39207  -0.58613       0.00000  -0.60730  -0.36077       0.00000  -0.55775  -0.43179

 Results of translations and rotations:

                                                   1                                 2                                 3
          Irreps                                  B1                                B2                                A1
     Frequencies                              1.7128                            1.2111                            1.2111
  Reduced masses                              6.0050                            6.0050                            6.0050
 Force constants                              0.0000                            0.0000                            0.0000
  IR intensities                              0.0000                            0.0000                            0.0000
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735
           2   1         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735
           3   1         0.57735   0.00000   0.00000       0.00000   0.57735   0.00000       0.00000   0.00000   0.57735

                                                   4                                 5                                 6
          Irreps                                  B1                                A2                                B2
     Frequencies                            -24.4784                          -70.3890                          -13.3187
  Reduced masses                              1.1261                            1.0080                            1.0461
 Force constants                             -0.0004                           -0.0029                           -0.0001
  IR intensities                            338.7926                            0.0000                          117.9659
        Atom  ZA               X         Y         Z             X         Y         Z             X         Y         Z
           1   8         0.08875   0.00000   0.00000       0.00000   0.00000  -0.00000       0.00000  -0.05042   0.00000
           2   1        -0.70432  -0.00000  -0.00000      -0.70711   0.00000   0.00000       0.00000   0.40010   0.58194
           3   1        -0.70432  -0.00000  -0.00000       0.70711   0.00000   0.00000       0.00000   0.40010  -0.58194


 *********************************************
 ***   Thermal Contributions to Energies   ***
 *********************************************

 Temperature               :       298.150000    Kelvin
 Pressure                  :         1.000000    Atm
 Molecular mass            :        18.015000    AMU
 Elec. total energy        :         0.000000    Hartree
 Scale factor of Frequency :         1.000000
 Rot. symmetry number      :         2
 The C2v  point group is used to calculate rotational entropy.

 Principal axes and moments of inertia in atomic units:
                                     1                   2                   3
     Eigenvalues --                  2.353178            4.421189            6.774367
           X                         0.000000            0.000000            1.000000
           Y                         1.000000            0.000000            0.000000
           Z                         0.000000            1.000000            0.000000

 Rotational temperatures            36.807165           19.590616           12.785522    Kelvin
 Rot. constants A, B, C             25.582291           13.616176            8.886393    cm**-1
                                   766.937800          408.202700          266.407362    GHz

 Thermal correction energies                              Hartree            kcal/mol
 Zero-point Energy                          :            0.020138           12.637066
 Thermal correction to Energy               :            0.022973           14.415863
 Thermal correction to Enthalpy             :            0.023917           15.008348
 Thermal correction to Gibbs Free Energy    :            0.002409            1.511792


 Job terminated correctly! Tue May 02 18:18:33 2017
